FixWhitespace.hs

```diff
@@ -141,6 +161,9 @@ main = do
       verbose = optVerbose opts
       config  = optConfig  opts
 
+  tabSize <- maybe (die "Error: Illegal TABSIZE, must be an integer.") return $
+    readMaybe $ optTabSize opts
+
```

Comment:
For consistency, checking whether the argument is a valid integer should be placed under `programOpts` (inside `case getOpt Permute options argv of ...`).

Reply:
Okay, I will do it later.